sdk/core/azure-core/src/main/java/com/azure/core/implementation/serializer/HttpResponseBodyDecoder.java

```diff
@@ -56,13 +58,13 @@
                         try {
                             final Object decodedErrorEntity = deserializeBody(bodyString,
                                 decodeData.getUnexpectedException(httpResponse.getStatusCode()).getExceptionBodyType(),
-                                null,
-                                serializer,
-                                SerializerEncoding.fromHeaders(httpResponse.getHeaders()));
+                                null, serializer, SerializerEncoding.fromHeaders(httpResponse.getHeaders()));
                             return decodedErrorEntity == null ? Mono.empty() : Mono.just(decodedErrorEntity);
                         } catch (IOException | MalformedValueException ignored) {
                             // This translates in RestProxy as a RestException with no deserialized body.
                             // The response content will still be accessible via the .response() member.
+                            logger.warning("Failed to deserialize the error entity. Error details: "
```

Comment:
You should pass the actual error in logger.warning as a parameter rather than manually printing out the message.

Reply:
Done.